.github/workflows/test.yml

```diff
@@ -66,8 +66,8 @@ jobs:
       fail-fast: false
       matrix: ${{ fromJSON(needs.list.outputs.matrix) }}
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-go@v2
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
         with:
           go-version: ${{ matrix.go }}
```

Comment:
How about adding `cache: true` here?

Reply:
Sounds good

Reply:
I don't know why, but actions/setup-go fails with Go 1.14 when `cache: true` is enabled.
https://github.com/go-sql-driver/mysql/actions/runs/3574840133/jobs/6010627696

Reply:
cache option in setup-go uses `go.sum` file. So we can not use it for now.